extensions.json

```
@@ -34,6 +34,11 @@
       "repository": "https://github.com/AlexCovizzi/vscode-sqlite",
       "version": "0.8.2"
     },
+    {
+      "id": "aliariff.vscode-erb-beautify",
+      "repository": "https://github.com/aliariff/vscode-erb-beautify",
+      "version": "0.2.0"
```

It seems safer to publish version `0.2.0` from the `v0.2.0` git tag instead of from the default git branch:

```suggestion
      "version": "0.2.0",
      "checkout": "v0.2.0"
```